django_mailbox/models.py

```diff
@@ -578,6 +611,23 @@ def html(self):
             self.get_email_object(), 'text', 'html'
         ).replace('\n', '').strip()
 
+    #rulzart - added a quick way to get Cc addresses
+    @property
+    def get_cc(self):
+        return self.get_email_object()["Cc"]
+
+    @property
+    def html_with_img(self):
+        """
+        Returns the message body matching content type 'text/html' with correct image src.
+        """
+        soup = BeautifulSoup(self.html)
+        for attachment in self.attachments.exclude(cid__isnull=True).exclude(cid__exact=''):
```

Comment:
I suggest move this filtering as ``QuerySet`` method. I think it's good to add support for prefetch filtered attachments in some way too.